JASP-Engine/JASP/R/anovarepeatedmeasures.R

```diff
@@ -1391,7 +1391,7 @@ AnovaRepeatedMeasures <- function(jaspResults, dataset = NULL, options) {
     marginalMeansTable$addFootnote(symbol = "\u2020", message = gettext("Bias corrected accelerated."))
     
   } else {
-    thisOverTitle <- gettextf("95%% CI for Mean Difference")
+    thisOverTitle <- gettext("95% CI for Mean Difference")
```

Comment:
Double `%` are still a necessity

Reply:
Yes, I just did that yesterday. I was believing in a miracle i think because in 
gettext("95% bca\u002A CI") anovarepeatedmeasure.R 1387 it works fine. I though that once you found that part of parsing that went wrong. Because this is very annoying too I think.

Reply:
See my explanation here: https://github.com/jasp-stats/INTERNAL-jasp/issues/777#issuecomment-597740225
It will only work when the translation and original text share the same "percent parameters". So it's just luck of the draw if it works.